alert-policies/traefik/HighHttp400.yml

```diff
@@ -0,0 +1,34 @@
+name: High 400 Error Rate
+
+description: |+
+  This alert is triggered when Traefik has high HTTP 4xx error rate service.
+type: STATIC
+nrql:
+  query: "FROM Metric 
+   SELECT filter(
+     sum(
+       traefik_service_requests_total), 
+     WHERE code Not RLIKE '4.*') * 100 
+   / filter(
+     sum(
+       traefik_service_requests_total), WHERE code RLIKE '.*') 
+   WHERE instrumentation.name = 'remote-write'"
```

Comment:
this needs an additional `FACET` to help users isolate the alerts

```
... FACET instance, service
```

Reply:
Thanks, resolved.